lib/exercism/client_version.rb

```diff
@@ -0,0 +1,21 @@
+module ExercismWeb
+  class ClientVersion
+    attr_accessor :user
+
+    def initialize(user:)
+      @user = user
+    end
+
+    def notice_when_client_outdated
+      unless dismissed?
+        "A new version of the exercism client is available (cli-2.4.0). Run <code>exercism upgrade</code> to get the latest hotness!"
```

Comment:
I'm hesitant to hard-code this. I'd rather either:

- get the most recent CLI release when the app boots up and store that in a constant that we can refer to, or
- have a `cli_versions` table and populate it based on what we get from the HTTP header when people submit (only storing each version once, and with a "first seen at" timestamp).

I think that the next CLI release will be a patch release, since changing where the CLI gets the text it displays is more of an implementation detail than a new feature.